### ISSUE TITLE
Variant display pages claims to just be showing Exome data but is showing Exome plus Genome data 

### ISSUE THREAD

**Issue by congenica-peter-fox** [broadinstitute/gnomad-browser#1591]
### What you did:
I looked at the GnomAD 4.1 data for variant chr2:79195912-G-T

https://gnomad.broadinstitute.org/variant/2-79195912-G-T?dataset=gnomad_r4

### What happened:
The foot of the Genetic Ancestry Group Frequencies table indicate that just exome data was being viewed

![Screenshot 2024-06-28 at 15 00 02](https://github.com/broadinstitute/gnomad-browser/assets/29795338/2ca9e161-f1d0-4c12-8915-fbb65218c9db)

However the total allele count from the Genetic Ancestry Group Frequencies table matches the allele count for the Exomes PLUS Genomes. Comparing the AN values with running a test sample through hail and the published GnomAD 4.1 data the allele counts do match the counts for **just** exomes.

**Comment by congenica-peter-fox** [comment]
Another page showing this behaviour is 

https://gnomad.broadinstitute.org/variant/2-79195912-G-A?dataset=gnomad_r4

**Comment by rileyhgrant** [comment]
Hey @congenica-peter-fox 

Thanks for filing this issue. I realize how this behavior in the browser is a bit confusing, as its combining the AN from the genomes data (while in that dataset the AC was 0). We started displaying AN from both datasets with the v4.1 release: https://gnomad.broadinstitute.org/news/2024-04-gnomad-v4-1/#joint-allele-numbers-at-all-sites-with-gnomad-variation

It's on our list to improve the UI here for clarity, as having just the exomes box checked is misleading.

**Comment by congenica-peter-fox** [comment]
Thanks, yeah I was quite confused. I'd built a hail pipeline using the GnomAD 4.1 data annotating with both exome and genome data. I'd then picked some sample variants with various test pattens such as multi-allelic or in both data sets or only in one. 

I was then trying to use the website to check my results. So when I looked at the total summary numbers at the top of the page it all made sense but when I compared the numbers I got from hail for each population against the website it just didn't line up. It took a while to figure out the population table was showing AN values across both exome and genome even though the variant didn't exist in one of those datasets and there was no way to change the selection.
